sample-apps/reviewer/app.py

```
@@ -0,0 +1,133 @@
+        self.review_config = self._load_review_config(studies, conf)
+
+        super().__init__(
+            app_dir=app_dir,
+            studies=self.review_config["studies"],
+            conf=conf,
+            name="MONAILabel Reviewer",
+            description="Lightweight review app - validates AI-generated segmentations",
+            version="1.0.0",
+        )
+
+        logger.info(f"MONAILabel Reviewer App initialized")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)